atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -149,6 +151,25 @@ def set_media_types(self, extra, worker_metadatas):
                 extra['image']['media_types'] = json.loads(annotations['media-types'])
                 return
 
+    def set_manifest_list_info(self, extra, worker_metadatas):
+        manifest_list_digests = self.workflow.postbuild_results.get(PLUGIN_GROUP_MANIFESTS_KEY)
+        # group_manifests returns either a list of manifest digests of the manifest lists,
+        # or a list of the image_manifests
+        if manifest_list_digests and not isinstance(manifest_list_digests[0], dict):
+            index = {}
+            index['tags'] = [image.tag for image in self.workflow.tag_conf.images]
+            repositories = self.workflow.build_result.annotations['repositories']['unique']
+            repo = ImageName.parse(repositories[0]).repo
+            # group_manifests added the registry, so this should be valid
+            for registry in self.workflow.push_conf.docker_registries:
+                pullspec = "{0}/{1}@{2}".format(registry.uri, repo,
+                                                manifest_list_digests[0].v2_list)
+                index['pull'] = [pullspec]
+                for tag in index['tags']:
+                    pullspec = "{0}/{1}:{2}".format(registry.uri, repo, tag)
+                    index['pull'].append(pullspec)
```

Comment:
We must not put "floating" tags here.  Ideally we should put the {version}-{release} tag here but no other tags, so it looks like:
pull: [ "registry.example.com/foo@sha256:...", "registry.example.com/foo:1.0-1" ]
tags: [ "1.0-1", "latest", ... ]

One way to arrange this is to use the knowledge that only the {version}-{release} tag may contain a dash character:
```
...
index['pull'] = [pullspec]
for tag in index['tags']:
    if '-' in tag:  # {version}-{release}
        index['pull'].append("{0}/{1}:{2}".format(registry.uri, repo, tag))
        break
```